test/performance/io/format_fastq_benchmark.cpp

```diff
@@ -0,0 +1,285 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+// @author: Clemens Cords <clemens.cords@fu-berlin.de>
+
+#include <benchmark/benchmark.h>
+#include <cstring>
+#include <fstream>
+
+#define SEQAN2_HAS_SEQAN3 __has_include(<seqan/seq_io.h>)
+#if SEQAN2_HAS_SEQAN3
+    #include <seqan/seq_io.h>
+#endif
+
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/alphabet/quality/phred42.hpp>
+#include <seqan3/io/sequence_file/format_fastq.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+#include <seqan3/range/view/to_char.hpp>
+#include <seqan3/test/performance/sequence_generator.hpp>
+#include <seqan3/std/filesystem>
+
+using namespace seqan3;
+
+unsigned int const SEED = 1234;
+
+// ============================================================================
+// generate fastq file
+// ============================================================================
+constexpr std::size_t SEQUENCE_LENGTH = 50; //length of nucleotide and quality sequence
```

Comment:
Please, replace all occurrences of `std::size_t` with `size_t` (without namespace)

We don't declare variables in all upper-case, please adjust it to lower case.

Reply:
fixed